src/screens/TopicPageScreen/index.js

```diff
@@ -415,34 +427,61 @@ const TopicPageScreen = (props) => {
         isFollow={isFollow}
         getTopicDetail={getTopicDetail}
       />
-      <TiktokScroll
-        ref={listRef}
-        contentHeight={dimen.size.TOPIC_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
-        data={feeds}
-        onEndReach={onEndReach}
-        onRefresh={onRefresh}
-        refreshing={loading}
-        renderItem={renderItem}
-        onScroll={handleScroll}
-        scrollEventThrottle={16}
-        showSearchBar={true}
-        searchHeight={isHeaderHide ? headerHideHeight : headerShowHeight}
-        snap
-        contentOffset={{x: 0, y: topPosition}}
-        contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
-        onMomentumScrollEnd={(event) => {
-          onWillSendViewPostTime(event, feeds);
-          fetchNextFeeds(event);
-        }}
-      />
+
+      <View
+        style={{
+          marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0
+        }}>
+        <TiktokScroll
+          ref={listRef}
+          contentHeight={dimen.size.TOPIC_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
+          data={feeds}
+          onEndReach={onEndReach}
+          onRefresh={onRefresh}
+          refreshing={loading}
+          renderItem={renderItem}
+          onScroll={handleScroll}
+          scrollEventThrottle={16}
+          showSearchBar={true}
+          snap
+          contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
+        />
+
+        <View
+          style={{
+            marginTop: isHeaderHide ? -dimen.size.TOPIC_FEED_HEADER_HEIGHT : 0,
+            minHeight: 548,
+            height: 700
+          }}>
+          <TiktokScroll
+            ref={listRef}
+            contentHeight={dimen.size.TOPIC_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
+            data={feeds}
+            onEndReach={onEndReach}
+            onRefresh={onRefresh}
+            refreshing={loading}
+            renderItem={renderItem}
+            onScroll={handleScroll}
+            scrollEventThrottle={16}
+            showSearchBar={true}
+            snap
+            contentInsetAdjustmentBehavior={feeds?.length > 1 ? 'automatic' : 'never'}
+            onMomentumScrollEnd={(event) => onWillSendViewPostTime(event, feeds)}
+          />
+        </View>
+      </View>
       <ButtonAddPostTopic topicName={topicName} onRefresh={onRefresh} />
       <BlockComponent
         ref={refBlockComponent}
         refresh={onBlockCompleted}
         refreshAnonymous={onDeleteBlockedPostCompleted}
         screen="topic_screen"
       />
-    </SafeAreaProvider>
+    </>
   );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this section seem to be related to the layout of the components. The `TiktokScroll` component is now wrapped in a `View` with a conditional margin top, and it appears twice, which seems like a mistake or a copy-paste error. Please review this part of the code to ensure it's correct.
```diff
- <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+ <>
...
- </SafeAreaProvider>
+ </>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->